ProjectSettings/ProjectSettings.asset

```diff
@@ -432,7 +432,8 @@ PlayerSettings:
   webGLUseEmbeddedResources: 0
   webGLUseWasm: 0
   webGLCompressionFormat: 1
-  scriptingDefineSymbols: {}
+  scriptingDefineSymbols:
+    14: 
```

Comment:
I'm not fully sure what Unity 5.5 is doing here, but anytime I build the WindowsStoreApp with the HoloToolkit build window, it tries to make this change.  Are other people not seeing this when they try to build the WindowsStoreApp?